transfer-archive/app/services/sas_service.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+from app.connectors.sas_generator.exceptions import SASTokenPreconditionFailed
+from app.connectors.sas_generator.sas_generator_client import SASGeneratorClient
+from app.settings import Settings
+
+logger = logging.getLogger(__name__)
+
+
+async def get_sas_generator_client() -> SASGeneratorClient:
+    return SASGeneratorClient(Settings().sas_generator_base_url)
+
+
+async def generate_sas_token(container_id: str, sas_generator_client: SASGeneratorClient):
```

Comment:
```suggestion
async def generate_sas_token(container_id: str, sas_generator_client: SASGeneratorClient) -> SASResponse:
```
(husk å importere `SASResponse` også